staging/src/k8s.io/apimachinery/pkg/util/errors/errors_test.go

```diff
@@ -262,6 +262,48 @@ func TestFlatten(t *testing.T) {
 	}
 }
 
+func TestUniqueAggregateWithCount(t *testing.T) {
+	testCases := []struct {
+		name     string
+		agg      Aggregate
+		expected Aggregate
+	}{
+		{
+			"input aggregate is nil",
+			nil,
+			nil,
+		},
+		{
+			"input aggregate is empty",
+			aggregate{},
+			aggregate{},
+		},
+		{
+			"input aggregate is singleton",
+			aggregate{fmt.Errorf("abc")},
+			aggregate{fmt.Errorf("abc(Count=1)")},
+		},
+		{
+			"input aggregate has duplicate errors",
+			aggregate{fmt.Errorf("abc"), fmt.Errorf("abc"), fmt.Errorf("ghi")},
+			aggregate{fmt.Errorf("abc(Count=2)"), fmt.Errorf("ghi(Count=1)")},
+		},
+		{
+			"input aggregate has duplicate and nested errors",
+			aggregate{aggregate{aggregate{fmt.Errorf("abc")}, fmt.Errorf("abc"), aggregate{fmt.Errorf("abc")}}},
+			aggregate{fmt.Errorf("abc(Count=3)")},
+		},
+	}
+	for i, testCase := range testCases {
+		t.Run(testCase.name, func(t *testing.T) {
+			agg := UniqueAggregateWithCount(testCase.agg)
+			if !reflect.DeepEqual(testCase.expected, agg) {
+				t.Errorf("%d: expected %v, got %v", i, testCase.expected, agg)
```

Comment:
Because you're using subtests, you don't need to include `%d` (`i`) in the output.